tests/toit_run_image/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2021 Toitware ApS.
+#
+# This library is free software; you can redistribute it and/or
```

Comment:
Should these CMakeList.txt files in the test directory be covered by LGPL or 0BSD?